### ISSUE TITLE
Failing property in PropCheck.Test.Movies

### ISSUE THREAD

**Issue by evnu** [alfert/propcheck#12]
With 35bfdaf and Elixir v1.5, the "server works fine" property in `test/movie_test.exs` fails here:

```
  1) property server works fine (PropCheck.Test.Movies)
     test/movie_test.exs:19
     Property server works fine failed. Counter-Example is:
     [[{:set, {:var, 1}, {:call, PropCheck.Test.MovieServer, :ask_for_popcorn, []}}]]
     
     code: nil
     stacktrace:
       test/movie_test.exs:19: (test)
```

**Comment by alfert** [comment]
This has apparently nothing to do with your changes but more with Elixir 1.5 - for whatever reason. I will take a  look at that.

**Comment by alfert** [comment]
Hmm, fails also in Elixir 1.4.5 under OTP 18 on travis.

**Comment by alfert** [comment]
Fixed in #15. I have no clue how this test has ever run, since starting the movie server GenServer was simply wrong.
